apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.types.ts

```diff
@@ -1,10 +1,14 @@
 import { z } from 'zod';
 
 import type { WithAIGradingStats } from '../../../ee/lib/ai-grading/types.js';
-import { AssessmentQuestionSchema, InstanceQuestionSchema } from '../../../lib/db-types.js';
+import {
+  AssessmentQuestionSchema,
+  DateFromISOString,
+  InstanceQuestionSchema,
+} from '../../../lib/db-types.js';
 
 export const InstanceQuestionRowSchema = InstanceQuestionSchema.extend({
-  modified_at: z.string(),
+  modified_at: DateFromISOString,
```

Comment:
This is unnecessary since `InstanceQuestionSchema` already uses this schema for this column.